### DISCUSSION TITLE
⚠ Implémentation concernant les QCM quand on se connecte

### DISCUSSION THREAD

**Discussion by ghost** [PapillonApp/Papillon-ED-Core#13] [Ideas]
# Contexte
Comme vous le savez, beaucoup de personnes menacent des établissements en piratant les comptes EcoleDirecte d'enseignants ou d'élèves. Aplim a ainsi développé très rapidement un système de sécurité, afin de pouvoir protéger plus facilement les comptes EcoleDirecte.

EDIT: Je réouvre, car même si j'ai déjà décrit l'implémentation (https://github.com/azgaresncf/ecoledirecte-api-docs?tab=readme-ov-file#-concernant-la-connexion-qcm). Ne serait-t-il pas judicieux de séparer chaque requête en différentes fonction du type getLoginQuestions() ou quelque chose dans le genre.

**Comment by camarm-dev** [comment]
Je pense la disscussion a été ouverte avant qu'on finisse l'implem...

C'est le cas aujourd'hui, comme décrit dans la doc les requêtes sont bien séparées 😉

![Screenshot_20240521-111928_GitHub.jpg](https://github.com/PapillonApp/Papillon-ED-Core/assets/77529508/193b6412-61f7-421b-9f93-d5860b21bb0e)
